src/rust/dbv2/migrations/2023-11-02-131654_small-queries/up.sql

```diff
@@ -0,0 +1,40 @@
+-- Your SQL goes here
+
+CREATE INDEX timeprice ON fill_events (time, price);
+
+CREATE FUNCTION api.price_info(market integer, seconds integer)
+RETURNS TABLE(last_price numeric, price_change numeric, high_price numeric, low_price numeric) AS $$
+    WITH market_data AS (
+        SELECT *
+        FROM market_registration_events
+        WHERE market_id = market
+    ),
+    fills AS (
+        SELECT *
+        FROM fill_events
+        WHERE market_id = market
+        AND time >= CURRENT_TIMESTAMP - '1 second'::interval * seconds
+        ORDER BY txn_version DESC, event_idx DESC
```

Comment:
This should now select from the fill events deduped table

Reply:
Nevermind, the migrations are not in the right order, and besides it doesn't make a difference